src/webcomponents/study/admin/study-settings-detail.js

```diff
@@ -0,0 +1,275 @@
+            //                         .value="${toolParams?.study}"
+            //                         .resource="${"STUDY"}"
+            //                         .opencgaSession="${this.opencgaSession}"
+            //                         .config="${{multiple: false, disabled: !!this.study}}"
+            //                         @filterChange="${e => this.onFieldChange(e, "study")}">
+            //                     </catalog-search-autocomplete>
+            //                 `,
+            //             },
+            //         }
+            //     ],
+            // },
+            {
+                // title: "Configuration Parameters",
+                id: "settings-json",
+                title: "Settings",
+                description: "TODO THINK ABOUT A DESCRIPTION",
+                display: {
+                    titleHeader: "h4",
+                    titleStyle: "margin: 5px 5px",
+                    descriptionClassName: "help-block",
+                    descriptionStyle: "margin: 0px 10px",
+                    // visible: () => !!this.config?.genotype?.type
+                },
+                elements: [
+                    {
+                        field: "body",
+                        type: "json-editor",
+                        display: {
+                            // rows: 25,
+                            // defaultLayout: "vertical"
+                        }
+                    },
+                ],
+            },
+            {
+                // title: "Configuration Parameters",
+                id: "settings-preview",
+                title: "Preview",
+                description: "TODO THINK ABOUT A DESCRIPTION",
+                display: {
+                    titleHeader: "h4",
+                    titleStyle: "margin: 5px 5px",
+                    descriptionClassName: "help-block",
+                    descriptionStyle: "margin: 0px 10px",
+                    // visible: () => !!this.config?.genotype?.type
+                },
+                elements: [
+                    {
+                        type: "custom",
+                        display: {
+                            defaultLayout: "vertical",
+                            render: () => {
+                                return html `
+                                    <!-- 2. Render Preview -->
+                                    <div>
+                                        <component-settings-previewer
+                                        .opencgaSession="${this.opencgaSession}"
+                                        .settings="${this.toolParams.body}">
+                                        </component-settings-previewer>
+                                    </div>
+                                `;
+                            }
+                        }
+                    },
+                ],
+            },
+        ],
+    });
+}
+*/
```

Comment:
Please remove if is not needed.

Reply:
I added a comment at the beginning. I am just keeping it for me, until I change the layout of of this content. I will remove it afterwards